app/models/voteable.rb

```diff
@@ -9,6 +9,11 @@ def upvote(user)
     @_vote.value = true
   end
 
+  def time_since_creation
```

Comment:
Looks great, Pat!  Could we also split out the days?  Maybe something like: 
     hours = hours_number % 24
     days = (hours_number / 24).round